libcst/_nodes/statement.py

```diff
@@ -948,6 +1024,75 @@ def _codegen_impl(self, state: CodegenState) -> None:
                 finalbody._codegen(state)
 
 
+@add_slots
+@dataclass(frozen=True)
+class TryStar(BaseCompoundStatement):
+    """
+    A ``try`` statement with ``except*`` blocks.
```

Comment:
github doesn't let me comment on a non-diff line 😠 

But - should we update the docstring of `Try`?